src/plugins/remark-code-import.ts

```diff
@@ -55,15 +54,13 @@ const remarkIncludeCode = () => {
           }
           // Inspired by astro aliases: https://docs.astro.build/en/guides/aliases/
           // Allows for specifying path from root directory
-          let fullPath = includePath.startsWith("@")
-            ? includePath.replace("@", "./")
-            : path.resolve(path.dirname(markdownFile.path), includePath);
+          let fullPath = includePath.startsWith("@") ? includePath.replace("@", "./") : includePath;
```

Comment:
This change was due to the error message. When resolve is used, the path gets a specific replacement (e.g. `/Users/joshka/local/ratatui-website/...` so it's impossible to test. It might be possible that I can pass a slight different parameter to markdownFile in the test that includes a rational path that would work (will experiment more)

This change doesn't seem to break anything though as all our current includes are @code prefixed.